x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/predicate/nulls/IsNotNullProcessor.java

```diff
@@ -15,7 +15,7 @@
     
     static final IsNotNullProcessor INSTANCE = new IsNotNullProcessor();
 
-    public static final String NAME = "inn";
+    public static final String NAME = "ninn";
```

Comment:
n=Null
inn=IsNotNull
---
ninn=NullIsNotNull

Not that it is suppose to be readable. Since the name is saved in the serialization, we try to keep them short with some ad-hoc convention.